src/components/nav.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "gatsby";
+import * as navStyles from './nav.module.scss';
+import "@fontsource/righteous";
+import "@fontsource/roboto";
```

Comment:
W tym projekcie jest akurat malo importów ale widzialem projekty gdzie jest np 40 importów, dobrze je wtedy kategoryzować poprzed dodaniie komendatrza np jak są fonty to przed importem można wrzucić linie komentarza //Fonts przed importem stylów //Styles...ale to tylko coś co kidyś widzialem ;)